testing-internal/build.gradle

```diff
@@ -1,6 +1,8 @@
 dependencies {
-    compile project(':testing')
+    compile project(':testing:junit')
+    compile project(':testing:junit4')
 
+    compile 'org.junit.jupiter:junit-jupiter-api'
```

Comment:
nit: `junit-jupiter-api` and `junit` seem unnecessary. How about removing them if so?